tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm

```diff
@@ -1139,11 +1151,17 @@ sub get_items_in_list {
     @items_list = $self->prettify_phenotypes(\@items_list, $species);
     @items_with_url = @items_list;
   }
+  elsif ($type eq 'disgenet_disease') {
+    foreach my $entry (@items_list) {
+      $entry =~ s/_/ /g;
+      push (@items_with_url, $entry);
+    }
+  }
   # Add external links
   else {
     foreach my $item (@items_list) {
       my $item_url = $item;
-      if ($type eq 'pubmed') {
+      if ($type eq 'pubmed' || $type eq 'disgenet_pmid') {
```

Comment:
For example rs797044835, the digenet_pmid are not links. Should they be?

Reply:
They should be. Can you check again? The branch was wrong.

Reply:
links now working.